controlplane/kubeadm/internal/filters_test.go

```diff
@@ -548,7 +548,7 @@ func TestMatchInitOrJoinConfiguration(t *testing.T) {
     },
     JoinConfiguration: nil,
     Files:             nil,
-    ... // 10 identical fields
+    ... // 11 identical fields
```

Comment:
I need to take some time to dig into impact of this change on existing clusters, because we need to make sure this change does not trigger rollouts
However, I'm not sure when I will get to it 😓

Reply:
Do you think it's feasible to have this merged before the next minor release?